packages/arb-token-bridge-ui/src/components/TransferPanel/USDCWithdrawal/USDCWithdrawalConfirmationDialog.tsx

```diff
@@ -0,0 +1,106 @@
+import { Tab, Dialog as HeadlessUIDialog } from '@headlessui/react'
+import { XMarkIcon } from '@heroicons/react/24/outline'
+
+import { Dialog, UseDialogProps } from '../../common/Dialog'
+import { Button } from '../../common/Button'
+import { ExternalLink } from '../../common/ExternalLink'
+import {
+  USDCBridgeInfo,
+  FastBridgeNames,
+  getFastBridges,
+  SpecialTokenSymbol
+} from '../../../util/fastBridges'
+import { TabButton } from '../../common/Tab'
+import { BridgesTable } from '../../common/BridgesTable'
+import { useNetworksAndSigners } from '../../../hooks/useNetworksAndSigners'
+import { getNetworkName, isNetwork } from '../../../util/networks'
+import { CommonAddress } from '../../../util/CommonAddressUtils'
+
+export function USDCWithdrawalConfirmationDialog(
+  props: UseDialogProps & { amount: string }
+) {
+  const { l1, l2 } = useNetworksAndSigners()
+
+  const { isArbitrumGoerli } = isNetwork(l2.network.id)
+
+  const from = l2.network
+  const to = l1.network
+  const fromNetworkName = getNetworkName(from.id)
+  const toNetworkName = getNetworkName(to.id)
+  const tokenSymbol = SpecialTokenSymbol.USDC
+
+  const fastBridges = getFastBridges({
```

Comment:
To be honest I'd just opt for not using `getFastBridges` and hardcoding the supported bridges within the dialog. The code in `getFastBridges` is quite messy and needs some refactoring